src/linear_algebra/finsupp.lean

```diff
@@ -413,6 +422,21 @@ end
 
 end lmap_domain
 
+section lcomap_domain
+
+variables {β : Type*} {R M}
+
+/-- Extend the domain of a finitely supported function by returning `0` outside the original domain.
```

Comment:
This restricts the domain of something (from `β` to `α`, that is essentially a subset of `β` via `f`).

Reply:
Here's an edit of the original docstring for `finsupp.comap_domain`:
```suggestion
/-- Given `f : α → β` and a proof `hf` that `f` is injective, `lcomap_domain f hf` is the linear map
sending  `l : β →₀ M` to the finitely supported function from `α` to `M` given by composing
`l` with `f`.
```